gcsfs/tests/test_core.py

```diff
@@ -1428,6 +1428,8 @@ def test_expiry_keyword():
     assert gcs.dircache.listings_expiry_time == 1
     gcs = GCSFileSystem(cache_timeout=1, token="anon")
     assert gcs.dircache.listings_expiry_time == 1
+    gcs = GCSFileSystem(cache_timeout=0, token="anon")
+    assert gcs.dircache.listings_expiry_time == 0
```

Comment:
Does this imply that the cache is disabled? What was the previous value before this PR, None?

Reply:
Yes, based on the docstring the `listings_expiry_time` should be set to 0, which means cache disabled w/ a timeout of 0. Before this change the `gcs.dircache.listings_expiry_time` evaluated to `None`